### PR TITLE
Add GOVERNANCE.md

### PR DESCRIPTION
**Pull request by bnb**
GOVERNANCE.md is a core file for any of the projects under the Open Governance model of the Node Foundation. This PR adds a GOVERNANCE.md file that dictates what open governance is for the Documenation WG.

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+* Technical direction
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of additional Collaborators
+
+For the current list of WG members, see the project
+README.md.
```
Comment (drewfish) [inline GOVERNANCE.md:15]: Hmmm... we don't appear to have a README.md.

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+### Collaborators
```
Comment (drewfish) [inline GOVERNANCE.md:17]: A bunch of this is also covered in CONTRIBUTING.md. Should we make an attempt to resolve/unify this doc and that one?

File: GOVERNANCE.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+Modifications of the contents of the Node.js Documentation WG repository are made on a collaborative basis. Anybody with a GitHub account may propose a modification via pull request and it will be considered by the project Collaborators. All pull requests must be reviewed and accepted by a Collaborator with sufficient expertise who is able to take full responsibility for the change. In the case of pull requests proposed by an existing Collaborator, an additional Collaborator is required participate and there is disagreement around a particular modification. See _Consensus Seeking Process_ below for further detail on the consensus model used for governance.
+
+Collaborators may opt to elevate significant or controversial modifications, or modifications that have not found consensus to the WG for discussion by assigning the ***WG-agenda*** tag to a pull request or issue. The WG should serve as the final arbiter where required.
```
Comment (drewfish) [inline GOVERNANCE.md]: I'm not sure what the `WG-agenda` tag means for this WG since we don't (currently) have any meetings.

File: GOVERNANCE.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+For the current list of Collaborators, see the project `README.md`.
+
+### WG Membership
```
Comment (drewfish) [inline GOVERNANCE.md:29]: Hmmm... this also seems to be talked about in CONTRIBUTING.md.

File: GOVERNANCE.md (hunk 2 above)
Comment (bnb) [inline GOVERNANCE.md]: Yes, I saw that the contributor additions were also noted in CONTRIBUTING.md

This is the only part that really seems to fully overlap is this:

>To join the team:
>
>If you'd like to join the documentation team, that's great! Note your interest on this issue. We'll get in contact with you shortly afterwards. Note that as part of the team, you are expected to be an exemplar of the project's values – both in making this specific project a welcoming place to contribute, as well as in the larger community (other issue trackers, conferences, etc.)
>
>Every month, new members will be added to the team. If you are added you will be paired with an existing editor from the team who will be responsible for your work for one month, upon successful completion of which you will become a full member.

Which end should be modified?

File: GOVERNANCE.md (hunk 3 above)
Comment (bnb) [inline GOVERNANCE.md]: I explain the use of the `wg-agenda` tag in the "WG Agenda" section.

File: GOVERNANCE.md (hunk 4 above)
Comment (bnb) [inline GOVERNANCE.md:29]: Yes, I saw that the contributor additions were also noted in CONTRIBUTING.md

This is the only part that really seems to fully overlap is this:

> To join the team:
> 
> If you'd like to join the documentation team, that's great! Note your interest on this issue. We'll get in contact with you shortly afterwards. Note that as part of the team, you are expected to be an exemplar of the project's values – both in making this specific project a welcoming place to contribute, as well as in the larger community (other issue trackers, conferences, etc.)
> 
> Every month, new members will be added to the team. If you are added you will be paired with an existing editor from the team who will be responsible for your work for one month, upon successful completion of which you will become a full member.

Which end should be modified?

File: GOVERNANCE.md (hunk 2 above)
Comment (bnb) [inline GOVERNANCE.md:17]: (Moved previous comment to correct section.) 

I'd say this is different, as the CONTRIBUTING.md talks about creation documentation, whereas this refers to the meta content of the WG.

File: GOVERNANCE.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
+### Documentation Working Group
```
Comment (chrisdickinson) [inline GOVERNANCE.md:1]: This should probably be a level-one heading.

File: GOVERNANCE.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+* Maintaining the list of additional Collaborators
+
+For the current list of WG members, see the project
+README.md.
+
+### Collaborators
+
+The Node.js Documenation WG GitHub repository is maintained by the WG and additional Collaborators who are added by the WG on an ongoing basis.
+
+Individuals making significant and valuable contributions are made Collaborators and given commit-access to the project. These individuals are identified by the WG and their addition as Collaborators is discussed over GitHub issues - at time of writing, these individuals are encouraged to comment on [this speific issue](https://github.com/nodejs/docs/issues/2).
```
Comment (chrisdickinson) [inline GOVERNANCE.md:21]: This is already covered in `CONTRIBUTING`.

**Comment by chrisdickinson** [comment]
Hm. I don't think we need this document quite yet. Much of `GOVERNANCE.md` — the code of conduct, how to become a contributor, and the issue tracker process are all included `CONTRIBUTING.md`, which GitHub handily links to from the "Create a PR" page.

With regards to the consensus seeking process, wg-agenda tag, and meetings — from my past experience with WGs I am not sure a weekly face to face meeting is worth the overhead of organizing and running it (which is significant, given the limited hours available to work on the project.) I'd prefer we gradually add process as necessary to address pain points. We're still in the bootstrapping phase right now, and so we haven't really found what those pain points are during normal operation. 

Perhaps we should commit to doing a status issue thread on the tracker to keep everyone informed of the state of the project?

**Comment by bnb** [comment]
@chrisdickinson Is that different than what I proposed in the `wg-agenda` section?

**Comment by bengl** [comment]
Closing this in favor of #46

**Pull request closed without merging**